docs/src/maintain/manage-releases.md

```diff
@@ -49,6 +49,47 @@ In rare cases, a second patch release might be necessary if the release is known
 
 After the patch release has been published (or no patch release is necessary), close the release issue and inform the team that they can start merging in semver-minor changes again.
 
+### Release Parameters
+
+The following tables shows examples of the option to select as `RELEASE_TYPE` when starting `eslint-js Release` (the `@eslint/js` package release) and `eslint Release` (the `eslint` package release) jobs on Jenkins to release a new version with the latest features. In both jobs, `main` should be selected as `RELEASE_BRANCH`.
+
+| **HEAD Version** | **Desired Next Version** | **`eslint-js Release`<br>`RELEASE_TYPE`** |
+| :---: | :---: | :---: |
+| `9.25.0` | `9.25.1` | `patch` |
+| `9.25.0` | `9.26.0` | `minor` |
+| `9.25.0` | `10.0.0-alpha.0` | `alpha.0` |
```

Comment:
Is this a typo or will we have a new `RELEASE_TYPE` `alpha.0`?

```suggestion
| `9.25.0` | `10.0.0-alpha.0` | `alpha` |
```

Reply:
We already have it in `eslint-js Release` (`@eslint/js` package release) job. It was necessary to bump major version (it passes `premajor` to `npm version` command, while `alpha`/`beta`/`rc` pass `prerelease`).

Reply:
I see, thanks! Never noticed that before.